experimental/list_formatter/Cargo.toml

```diff
@@ -42,8 +42,8 @@ path = "src/lib.rs"
 
 [features]
 std = ["icu_provider/std", "icu_locid/std"]
-default = ["provider_serde", "icu4x_human_readable_de"]
+default = ["icu4x_human_readable_de"]
```

Comment:
Note: provider_serde is a default feature everywhere else.  We have other issues though to track this decision.

Reply:
`icu4x_human_readable_de` enables `provider_serde`